### ISSUE TITLE
"Toggle" is difficult to translate in many languages and should be avoided

### ISSUE THREAD

**Issue by afercia** [WordPress/twentytwentyone#526]
**Describe the bug**
Minor. Not urgent. Can be addressed at any time before strings freeze,

The buttons to open the navigation sub-menus us a visually hidden text "Toggle child menu".

The verb "toggle" is difficult to translate in many languages and should preferably be avoided. Trac reference with more details:

**Remove use of "Toggle" in strings**
https://core.trac.wordpress.org/ticket/34753

Also, considering speech recognition software users, it would be great to simplify the buttons text and find one that is very intuitive. Rationale:

- speech recognition software users see a "plus" icon
- to activate the button they need to know what the accessible name of the button (the button text) is
- a voice command like "Click plus" won't work of course
- as a speech recognition software user I would then try something like "click open" or "click open menu" 
- I'm not sure I would every try something like "click open child menu" as that would be not intuitive
- overall I'd consider to use "Open menu": even it _technically_ not 100% correct, I'd tend to think it's the most intuitive text to use 
- for screen reader users, the buttons would be announced in a way that's still meaningful:
  - Open menu, collapsed
  - Open menu, expanded

**Comment by melchoyce** [comment]
> overall I'd consider to use "Open menu": even it technically not 100% correct, I'd tend to think it's the most intuitive text to use

This makes sense to me 👍
